glusterd2/main.go

```diff
@@ -108,6 +109,10 @@ func main() {
 		log.WithError(err).Fatal("Failed to generate local auth token")
 	}
 
+	if err := cluster.LoadClusterAttributes(); err != nil {
```

Comment:
Not really required. since all the default values and other details available with Glusterd2 itself.